tests/cases/mount/mount.jsx

```diff
@@ -0,0 +1,111 @@
+import {findOne} from "../../../dom/traverse";
+import QUnit from "qunit";
+import {mount} from "../../../mount";
+import {h, Component} from "preact";
+
+
+QUnit.module("mount()", {
+    beforeEach: () =>
+    {
+        findOne("#qunit-fixture").innerHTML = `
+            <div id="fixture-1" class="fixture"></div>
+            <div id="fixture-2" class="fixture"></div>
+            <div id="fixture-3" class="fixture"></div>
+        `;
+    },
+});
+
+QUnit.test(
+    "mount() test that the correct parameters are passed, fixes #158",
+    (assert) =>
+    {
+        let context = findOne("#qunit-fixture");
+        let counter = 0;
+
+        let expectedArguments = [
+            {
+                0: findOne("#fixture-1", context),
+                1: 42,
+            },
+            {
+                0: findOne("#fixture-2", context),
+                1: 42,
+            },
+            {
+                0: findOne("#fixture-3", context),
+                1: 42,
+            },
+        ];
+
+        let getExpectedArguments = () => {
+            let args = expectedArguments[counter];
+            counter++;
+            return args;
+        };
+
+        class ExampleMountComponent
+        {
+            constructor (firstParam, secondParam)
+            {
+                assert.deepEqual(arguments, getExpectedArguments())
+            }
+
+            init () {}
+        }
+
+        mount(".fixture", ExampleMountComponent, {params: [42], context});
+    }
+);
+
+QUnit.test(
+    "mount() with JSX component",
```

Comment:
Can we get a test-case for a functional JSX component as well?

Reply:
Was already added.